### ISSUE TITLE
Spurious macOS version error

### ISSUE THREAD

**Issue by fonnesbeck** [apple/tensorflow_macos#182]
When running the install script as described on the README on a MacBook Air M1 and macOS 11.2.1, I get the following error:

`ERROR: TensorFlow with ML Compute acceleration is only available on macOS 11.0 and later.`

Clearly this is a false error message. Anyone else see this?

**Comment by fonnesbeck** [comment]
Looking at what the script is looking for, I'm getting the following for OS version:

`echo $(sw_vers -productVersion)
`10.16`

Not sure why the OS version is incorrect.

**Comment by fonnesbeck** [comment]
Am I really the only one seeing this? Still does not work in alpha3.

**Comment by sqzhang-jeremy** [comment]
> Am I really the only one seeing this? Still does not work in alpha3.
Nope. I've met the same error bug with you, but  on a MacBook pro M1 and macOS 11.1. 
I closed the conda env and redid the command:bash download_and_install.sh, it worked.

**Comment by fonnesbeck** [comment]
Thanks for the suggestion; it does not work even after deactivating the conda env. The issue with the OS version remains--it still thinks I am running OS 10.16.

**Comment by sqzhang-jeremy** [comment]
> Thanks for the suggestion; it does not work even after deactivating the conda env. The issue with the OS version remains--it still thinks I am running OS 10.16.

1)Reboot ur mac.
2)Check the right python version you've installed. 
I've referenced an article: https://blog.csdn.net/knowledge_endlesss/article/details/113853831, which gives a very clear thinking to install tf/torch on Macbook M1. Now only version 3.9 and above supports ARM on the Python website, and tensorflow_macos only supports python3.8+. Therefore, a >=3.9 python should be installed first. Then, conda manger like miniforge shoulde be created a python=3.8 env. 
3)redo the bash download_and_install.sh.

Hope it will work. There are many bugs in tensorflow_macos, maybe I was lucky.

**Comment by fonnesbeck** [comment]
The only way I could get around it was to strip out all of the version checks in the scripts, download the installer, strip out the version check in the installer script, and then run it. 

The version check clearly does not work, and should be removed or modified to work correctly with Big Sur.
